Automattic/wp-calypso
# Hosting Metrics: Change the bar chart tooltips text

- Related to https://github.com/Automattic/dotcom-forge/issues/3356

## Proposed Changes

* Change Bar Chart Tooltip texts.


<img width="1264" alt="CleanShot 2023-08-10 at 14 20 07@2x" src="https://github.com/Automattic/wp-calypso/assets/36432/0d0c7e0b-1914-4d73-90b4-c6255e4c4a09">


## Testing Instructions



* Navigate to `/site-monitoring/${siteSlug}`
* Scroll down
* Put the mouse over the bar chart legend status code
* Observe the new texts in the tooltips

### Video

https://github.com/Automattic/wp-calypso/assets/779993/066a2fbb-8964-41a3-8e9d-40f166fd877c


## Pre-merge Checklist
